app/wyze_bridge.py

```diff
@@ -71,8 +71,10 @@ def setup_streams(self):
             self.add_substream(cam, options)
             stream = WyzeStream(cam, options)
             stream.rtsp_fw_enabled = self.rtsp_fw_proxy(cam, stream)
+            if cam.is_kvs:
+                self.api.setup_mtx_proxy(cam_name=cam.name_uri, uri=stream.uri)
 
-            self.mtx.add_path(stream.uri, not options.reconnect)
+            self.mtx.add_path(stream.uri, not options.reconnect, cam.is_kvs)
```

Comment:
Should this be moved into the above `if cam.is_kvs` statement? It's likely I'm misunderstanding something here, though :slightly_smiling_face: 
```suggestion
if cam.is_kvs:
    self.api.setup_mtx_proxy(cam_name=cam.name_uri, uri=stream.uri)
    self.mtx.add_path(stream.uri, not options.reconnect, cam.is_kvs)
```

Reply:
They are doing a couple different things, the setup call is calling the wyze API to get the KVS setup started in the background for the proxy. The add_path call needs the `is_kvs` bool so it knows that it has to add a source path to MTX for the proxy instead of using the event bus.